app/home/routes.py

```diff
@@ -3,25 +3,16 @@
 Copyright (c) 2020 - DevsBranch
 """
 import os
-import shutil
 import json
 from pathlib import Path
 from datetime import date
-from datetime import datetime
 from flask import render_template, redirect, url_for, request, current_app, flash
 from flask_login import login_required, current_user
 from jinja2 import TemplateNotFound
-
-from app import db
-from app.base.forms import PropertyForm, UpdateAccountForm
+from app.base.forms import CreatePropertyForm, UpdatePropertyForm
 from app.base.models import Property
 from app.home import blueprint
-from app.base.file_handler import (
-    property_image_handler,
-    save_profile_picture,
-    create_images_folder,
-)
-
+from app.base.file_handler import save_images_to_temp_folder
 ALLOWED_IMG_EXT = [".png", ".jpg", ".jpeg"]
```

Comment:
since  you are taking care of this validation in the form
I guess you could remove it from here?

Reply:
I have added a validation method to the forms